TEST.md

```diff
@@ -84,6 +84,7 @@ cluster. Prior to running the test suite, verify that:
 
 - The Conduit docker images you're trying to test have been built and are
   accessible to the Kubernetes cluster to which you are deploying
+- `kubectl version` reports the client version as v1.11.1 or later.
```

Comment:
I use gcloud to manage my kubectl installation, and gcloud's version of kubectl is at v1.9.7 currently ([release notes](https://cloud.google.com/sdk/docs/release-notes#kubernetes_engine_4)). I have a strong preference for our minimum required version to not exceed the version that's available via gcloud.

Reply:
> I have a strong preference for our minimum required version to not exceed the version that's available via gcloud.

What's the rationale?

Reply:
> What's the rationale?

Never mind.

I just installed kubectl through gcloud and the version is "v1.11.0."

Reply:
My rationale is that we shouldn't require the latest available version of kubectl to delete a few namespaces; I would rather update the script so that it works regardless of kubectl version.

I think something like this would work:

```
if ! namespaces=$(kubectl get ns -oname | grep "/$linkerd_namespace"); then
  echo "no namespaces found for [$linkerd_namespace]"
  exit 1
else
  kubectl delete $namespaces
fi
```

That will exit with an error if no namespaces exist, and otherwise it will delete all namespaces in parallel.